docs/usage/prune.rst

```diff
@@ -34,6 +34,5 @@ first so you will see what it would do without it actually doing anything.
 
 There is also a visualized prune example in ``docs/misc/prune-example.txt``:
 
-.. highlight:: none
```

Comment:
guess if you just kill that, it will apply ... the python highlighter (or bash?).

Reply:
ah, of course. I only looked at the man page results and not at the html docs.

Reply:
I've come to the conclusion I don't have enough experience with Sphinx to fix the highlight issue, at least the other minor fix can be merged